# ====== DO NOT MODIFY! AUTOGENERATED FROM AN ACTION DEFINITION ======
#define the goal, srs_msgs/srs_action[] action_sequence	#Specify what action sequence to be applied
string action
string parameter
